portfolio/src/main/webapp/index.html

```diff
@@ -8,8 +8,10 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
+      <img src="images/Headshot.PNG" alt="Picture of Frederick Santiago" width="200" height="300">  
```

Comment:
Let's use lowercase for all our resource files. While this not strictly required, using lowercase everywhere avoids us from making mistakes while linking to pages/files. For example, using capital first letter in the file name but not in the link can lead to "resource not found" errors. This also avoids similar errors in case the user is hand-typing the URL.